.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
We should add `.vscode` to our `.gitignore` file to keep it from being tracked by Git. This prevents these settings from overwriting a collaborator's local settings.